ee/clickhouse/queries/trends/util.py

```diff
@@ -98,16 +99,22 @@ def get_active_user_params(filter: Filter, entity: Entity, team_id: int) -> Dict
     return params
 
 
-def populate_entity_params(entity: Entity, table_name: str = "") -> Tuple[Dict, Dict]:
+def populate_entity_params(
+    entity: Entity, team_id: int, table_name: str = "", *, with_prop_clauses: bool = False
```

Comment:
`clauses` is an odd name. What about `with_prop_filters`?

Also is the name of the function still accurate? Maybe it should be `filter_by_entity`?

Reply:
I went for existing terminology (`parse_prop_clauses()`), but may make sense to rename this.

Reply:
`filter_by_entity` sounds a bit too imperative to me, went for `get_entity_filtering_params` for now.